libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryUtils.kt

```diff
@@ -48,10 +59,16 @@ fun obtainAbsoluteDistance(
     } ?: return 0
 }
 
+/**
+ * Provide last point of [DirectionsRoute]
+ */
 fun obtainRouteDestination(route: DirectionsRoute?): Point =
     route?.legs()?.lastOrNull()?.steps()?.lastOrNull()?.maneuver()?.location()
         ?: Point.fromLngLat(0.0, 0.0)
 
+/**
+ * Provide volume level in range *0..100*
```

Comment:
```suggestion
 * Provide the volume level in the range of *0..100*
```

Reply:
0-100 what? What's the unit?

Reply:
it should be percentage. wdyt?
```suggestion
 * Provide the volume level in the percentages(range is *0..100*)
```

Reply:
@langsmith ^^